Src/libCZI/CziReaderWriter.cpp

```diff
@@ -306,16 +307,16 @@ void CCziReaderWriter::Finish()
         attchmntDirWriteInfo.segmentPosForNewSegment = this->nextSegmentInfo.GetNextSegmentPos();
         attchmntDirWriteInfo.entryCnt = this->attachmentDirectory.GetEntryCnt();
         attchmntDirWriteInfo.enumEntriesFunc = [&](const std::function<void(size_t, const CCziAttachmentsDirectoryBase::AttachmentEntry&)>& f)->void
-        {
-            this->attachmentDirectory.EnumEntries(
-                [&](size_t index, const CCziAttachmentsDirectoryBase::AttachmentEntry& e)->bool
-                {
-                    f(index, e);
-            return true;
-                });
-        };
+            {
+                this->attachmentDirectory.EnumEntries(
+                    [&](size_t index, const CCziAttachmentsDirectoryBase::AttachmentEntry& e)->bool
+                    {
+                            f(index, e);
+                            return true;
```

Comment:
Indention now looks somewhat strange here? -> It's 8 spaces, but should be only 4?

Reply:
right, this looked strange - auto-formatted this file, looks ok now.